stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_account_preset_reason" model="ir.ui.view">
+        <field name="name">Preset Reason Accounts</field>
+        <field name="model">stock.inventory.line.reason</field>
+        <field name="inherit_id" ref="stock_change_qty_reason.view_preset_reason"/>
+        <field name="arch" type="xml">
+            <field name="description" position="after">
+                <field name="account_reason_input_id"/>
```

Comment:
@hveficent Maybe these fields should be visible for account people only ?

Reply:
now visible for account users at least

Reply:
How? Do not see changes

Reply:
Now they are, missed them on previous push